newIDE/app/src/VariablesList/VariableRow.js

```diff
@@ -90,17 +92,22 @@ const VariableRow = ({
       {arrayElement ? (
         <Text noMargin>{name}</Text>
       ) : (
-        <TextField
+        <SemiControlledAutoComplete
           margin="none"
-          style={{
-            fontStyle: origin !== 'inherited' ? 'normal' : 'italic',
```

Comment:
We probably want to keep this style, please put a TODO comment otherwise we're risking silently losing a "feature".